overviewOfAstraDB/README.md

```diff
@@ -1,5 +1,5 @@
 # Overview
-Understand Astra DB and how it differs from its underlying technology, Apache Cassandra®.
+Astra DB is a serverless database service (DBaaS) built on Apache Cassandra®. 
```

Comment:
Why not specify it in full like `..database-as-a-service...(DBaaS)...`?

Reply:
Also please remove the extra space after the period that got introduced here.

Reply:
We could potentially do that @msmygit, seems fine to me. @simonfraserduncan, do you have any thoughts on this?

Reply:
So, something like:
"Astra DB is a serverless database-as-a-service (DBaaS) built on Apache Cassandra®."

Reply:
I think that's fine. Many evaluating Astra will be familiar with DBaaS as an acronym, and for those who aren't, we're providing the explanation (which is more than what we offer on https://www.datastax.com/products/datastax-astra right now).

Reply:
Thank you